vud_widget.libraries.yml

```diff
@@ -0,0 +1,11 @@
+plain:
```

Comment:
Rename the file to vud, which is the related module name, otherwise it will not register.